src/Juvix/Compiler/Backend/VampIR/Pretty/Keywords.hs

```diff
@@ -0,0 +1,38 @@
+module Juvix.Compiler.Backend.VampIR.Pretty.Keywords where
+
+import Juvix.Compiler.Backend.VampIR.Language
+import Juvix.Data.CodeAnn
+import Juvix.Extra.Strings qualified as Str
+
+kwDef :: Doc Ann
```

Comment:
Since we introduced `Data.Keyword`, it would make sense to use it here. It will make these definitions more reusable (e.g. in a parser).
Feel free to ignore this if you think otherwise

Reply:
I adapted this from analogous code for GEB. Maybe it would make sense to refactor the GEB code, because there we have both keywords based on Data.Keyword and ones for printing implemented like here.

But for VampIR I think using Data.Keyword is an overkill. We will not be parsing VampIR input, only generating it, and most fields in Keyword are redundant. We essentially only need a name for a keyword string.